packages/google_maps_flutter/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/TileOverlaysController.java

```diff
@@ -0,0 +1,108 @@
+package io.flutter.plugins.googlemaps;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.TileOverlay;
+import com.google.android.gms.maps.model.TileOverlayOptions;
+import io.flutter.plugin.common.MethodChannel;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+class TileOverlaysController {
+
+  private final Map<String, TileOverlayController> tileOverlayIdToController;
+  private final MethodChannel methodChannel;
+  private GoogleMap googleMap;
+
+  TileOverlaysController(MethodChannel methodChannel) {
+    this.tileOverlayIdToController = new HashMap<>();
+    this.methodChannel = methodChannel;
+  }
+
+  void setGoogleMap(GoogleMap googleMap) {
+    this.googleMap = googleMap;
+  }
+
+  void addTileOverlays(List<Object> tileOverlaysToAdd) {
+    if (tileOverlaysToAdd == null) {
+      return;
+    }
+    for (Object tileOverlayToAdd : tileOverlaysToAdd) {
+      addTileOverlay(tileOverlayToAdd);
+    }
+  }
+
+  void changeTileOverlays(List<Object> tileOverlaysToChange) {
+    if (tileOverlaysToChange == null) {
+      return;
+    }
+    for (Object tileOverlayToChange : tileOverlaysToChange) {
+      changeTileOverlay(tileOverlayToChange);
+    }
+  }
+
+  void removeTileOverlays(List<Object> tileOverlayIdsToRemove) {
```

Comment:
Can just be `List<String>`, ditto above methods

Reply:
I did in the same style as the existing code